components/Receive/components/NewAddress.tsx

```diff
@@ -64,11 +60,11 @@ const NewAddress: React.FunctionComponent<NewAddressProps> = ({
       } else {
         newAddressStr = await RPCModule.createNewTransparentAddressProcess();
       }
-      console.log(newAddressStr);
+
       if (newAddressStr) {
-        if (newAddressStr.toLowerCase().startsWith(GlobalConst.error)) {
+        if (newAddressStr.toLowerCase().includes(GlobalConst.error)) {
           console.log(`Error new address ${newAddressStr}`);
-          //return newAddressStr;
+          return newAddressStr;
```

Comment:
here it needs to show a snackbar to the user with the error from zingolib... this return do nothing...